wallet/src/account/output_cache/mod.rs

```diff
@@ -42,13 +42,15 @@ use wallet_types::{
 pub struct OutputCache {
     txs: BTreeMap<AccountWalletTxId, WalletTx>,
     consumed: BTreeSet<UtxoOutPoint>,
+    pools: BTreeMap<PoolId, (UtxoOutPoint, BlockInfo)>,
```

Comment:
Would it make sense to turn the tuple into a proper struct? It could define some convenience methods to access the components of `BlockInfo`.

Reply:
As the only use is for printing, I will wrap it in a struct at the CLI level with a custom Display for a better message to the user.